src/cavendish_particle_tracks/_decay_angles_dialog.py

```diff
@@ -116,7 +168,7 @@ def _setup_decayangles_layer(self):
 
         text = {
             "string": ["Λ", "p", "π"],
-            "size": 14,
+            "size": 20,
```

Comment:
Again, this looks like scope creep. Is the text currently too small?

Reply:
Yes.